### ISSUE TITLE
MultiStepLR milestones extremely high

### ISSUE THREAD

**Issue by prattcmp** [bshall/Tacotron#3]
I noticed while training this model that the learning rate never changes. When I look at the milestone hparams, they are extremely high, starting at 20000. [MultiStepLR](https://pytorch.org/docs/stable/optim.html) looks for an array of epoch milestones, but it looks like you have training step milestones. 

Is this how you actually trained the model? Am I missing some other component to this?

**Comment by prattcmp** [comment]
For anyone else who is wondering, @bshall set the scheduler to step every iteration, not every epoch loop. I assume this was so it'd be easier to align with the stepping parameters used in the paper.

https://github.com/bshall/Tacotron/blob/6fee34a7c3a9d4ceb9215ed3063771a9287010e1/train.py#L142

I wonder if there's any overhead for `scheduler.step()` that may slow down training.
